Source/WebKit/WebKit.xcodeproj/project.pbxproj

```diff
@@ -21871,9 +21877,7 @@
 		};
 		58E7CD782E78575C00338ED0 /* PBXTargetDependency */ = {
 			isa = PBXTargetDependency;
-			platformFilters = (
-				ios,
-			);
+			platformFilter = ios;
```

Comment:
I assume Xcode automatically made this change? It's probably fine to keep, if so.